.github/workflows/build_and_push_to_docker.yml

```diff
@@ -30,23 +72,129 @@ jobs:
         password: ${{ secrets.DOCKER_PASSWORD }}
 
     - uses: actions/checkout@v2
-    - name: Build renku project docker images
+    - name: Build renku project python-based docker images
       run: |
         if [[ ${{ github.ref }} == refs/tags* ]]; then
           export LABEL=$(echo ${{ github.ref }} | cut -d / -f 3)
         else
           export LABEL=$(echo ${{ github.sha }} | cut -c 1-7)
         fi
+        if [ -z $RENKU_VERSION ]; then
+          export RENKU_PIP_SPEC=renku
```

Comment:
It's not necessary to set `RENKU_PIP_SPEC` because it's just taking whatever was set in the base image. See the warning here: https://github.com/SwissDataScienceCenter/renkulab-docker/runs/570365713?check_suite_focus=true#step:4:430